### ISSUE TITLE
Ciao-controller: create a k8s cluster operations

### ISSUE THREAD

**Issue by kaccardi** [01org/ciao#713]
includes creating/deleting/updating the cluster. Create the tenant, tenant storage pool, and launching the master and worker workloads for the cluster.

**Comment by kaccardi** [comment]
create a new tenant in keystone per cluster. Allocate a ceph storage pool per tenant. launch the k8s master, followed by the specified number of k8s workers.

**Comment by kaccardi** [comment]
gophercloud does not currently have support for the identity v3 api for projects. This support will need to be added to gophercloud or provided outside of gophercloud.

**Comment by kaccardi** [comment]
depends on #720 
depends on #725

**Comment by kaccardi** [comment]
we changed our k8s strategy, so this is no longer relevant.
